pulp.spec

```diff
@@ -264,6 +264,7 @@ Group: Development/Languages
 Requires: python-%{name}-common = %{pulp_version}
 Requires: python-celery >= 3.1.0
 Requires: python-celery < 3.2.0
+Requires: python-krbV
```

Comment:
I believe we should not require this package, but instead we should document that we can use it if the user installs it. However, I don't feel that strongly about it. @bmbouter, what do you think?